cirq/google/optimizers/convert_to_sycamore_gates_test.py

```diff
@@ -223,3 +225,18 @@ def test_convert_to_sycamore_equivalent_unitaries(gate):
     u1 = cirq.unitary(cirq.Circuit(converted))
     u2 = cirq.unitary(operation)
     cirq.testing.assert_allclose_up_to_global_phase(u1, u2, atol=1e-8)
+
+
+def test_convert_to_sycamore_tabulation():
+    # A tabulation for the sycamore gate with an infidelity of .1.
+    sycamore_tabulation = gate_product_tabulation(cirq.unitary(cirq.google.SYC),
+                                                  .1)
```

Comment:
I would specify a random_state argument to make sure the test is deterministic.

Reply:
Good call, done.